datameta/settings.py

```diff
@@ -21,19 +21,19 @@ def get_setting(db, key):
 def get_setting_value_type(setting):
     value_type = None
 
-    if setting.int_value != None:
+    if setting.int_value:
```

Comment:
I just realised, in this case this would cause issues with null values `0, 0.0, "", ...`, the `is not None` would be the better solution. Sorry!
```suggestion
    if setting.int_value is not None:
```